circle.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   cache_directories:
     - "~/cache"
   override:
-    - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev
+    - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev libsasl2-dev
```

Comment:
Our Dockerfile promises that Girder + all core plugins will install/function correctly, so if that library is necessary then we have to.